src/pybind/mgr/dashboard/controllers/feedback.py

```diff
@@ -0,0 +1,34 @@
+# # -*- coding: utf-8 -*-
+from typing import Optional
+
+from .. import mgr
+from ..services import feedback
+from . import ApiController, ControllerDoc, RESTController
+from ..model.feedback import Feedback
+from ..security import Scope
+
+@ApiController('/feedback', Scope.CONFIG_OPT)
+@ControllerDoc("Feedback API", "Report")
+class FeedbackController(RESTController):
+    issueAPIkey = None
+
+    def __init__(self):  # pragma: no cover
+        super(FeedbackController, self).__init__()
+        self.cephTrackerClient = feedback.CephTrackerClient()
+
+    def create(self, project_id, tracker_id, subject, description, severity: Optional[str] = '3 - Minor'):
+        """
+        Create an issue.
+        :param component: The buggy ceph component.
+        :param title: The title of the issue.
+        :param description: The description of the issue.
+        """
+        feedback = Feedback(project_id, tracker_id, subject, description, severity)
+        return self.cephTrackerClient.create_issue(feedback)
```

Comment:
```suggestion
        try:
            new_issue = Feedback(project_id, tracker_id, subject, description, severity)
        except ValueError as error:
            raise DashboardException(msg=str(error), component='feedback')
        return self.cephTrackerClient.create_issue(new_issue)
```
1. Avoid variable name shadowing (feedback from an import).
2. return error 400 (BadRequest) on ValueError.